perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1622,6 +1628,7 @@ sub process_garden_data : Private {
         $data->{bin_count} = 1;
         $data->{new_bins} = 1;
         my $cost_pa = $c->cobrand->garden_waste_sacks_cost_pa();
+        $cost_pa = $c->cobrand->apply_garden_waste_discount($cost_pa) if $data->{apply_discount};
```

Comment:
This one is missing brackets round LHS so would be set to size of returned array rather than the cost, I think. One to fix before merging.